src/mlpack/methods/preprocess/preprocess_describe_main.cpp

```diff
@@ -0,0 +1,226 @@
+    "dataset.csv, and keep the default settings, we could run"
+    "\n\n"
+    "$ mlpack_preprocess_describe -i dataset.csv -v"
+    "\n\n"
+    "If we want to customize the width to 10 and precision to 5 and consider "
+    "the dataset as a population, we could run"
+    "\n\n"
+    "$ mlpack_preprocess_describe -i dataset.csv -w 10 -p 5 -P -v");
+
+// Define parameters for data.
+PARAM_STRING_IN_REQ("input_file", "File containing data,", "i");
+PARAM_INT_IN("dimension", "Dimension of the data. Use this to specify a "
+    "dimension", "d", 0);
+PARAM_INT_IN("precision", "Precision of the output statistics.", "p", 4);
+PARAM_INT_IN("width", "Width of the output table.", "w", 8);
+PARAM_FLAG("population", "If specified, the program will calculate statistics "
+    "assuming the dataset is the population. By default, the program will "
+    "assume the dataset as a sample.", "P");
+PARAM_FLAG("rowMajor", "If specified, the program will calculate statistics "
+    "assuming the dataset is organized in row major. By default, the program "
+    "will assume the dataset is a column major.", "r");
+
+/**
+* Calculates the sum of deviations to the Nth Power.
+*
+* @param input Vector that captures a dimension of a dataset.
+* @param rowMean Mean of the given vector.
+* @param n Degree of power.
+* @return sum of nth power deviations.
+*/
+double SumNthPowerDeviations(const arma::rowvec& input,
+    const double& fMean,
+    size_t n) // Degree of Power
+{
+  return arma::sum(arma::pow(input - fMean, static_cast<double>(n)));
+}
+
+/**
+ * Calculates Skewness of the given vector.
+ *
+ * @param input Vector that captures a dimension of a dataset
+ * @param rowStd Standard Deviation of the given vector.
+ * @param rowMean Mean of the given vector.
+ * @return Skewness of the given vector.
+ */
+double Skewness(const arma::rowvec& input,
+    const double& fStd,
+    const double& fMean,
+    const bool population)
+{
+  double skewness = 0;
+  const double S3 = pow(fStd, 3);
+  const double M3 = SumNthPowerDeviations(input, fMean, 3);
+  const double n = input.n_elem;
+  if (population)
+  {
+    // Calculate Population Skewness
+    skewness = M3 / (n * S3);
+  }
+  else
+  {
+    // Calculate Sample Skewness
+    skewness = n * M3 / ((n - 1) * (n - 2) * S3);
+  }
+  return skewness;
+}
+
+/**
+ * Calculates excess kurtosis of the given vector.
+ *
+ * @param input Vector that captures a dimension of a dataset
+ * @param rowStd Standard Deviation of the given vector.
+ * @param rowMean Mean of the given vector.
+ * @return Kurtosis of the given vector.
+ */
+double Kurtosis(const arma::rowvec& input,
+    const double& fStd,
+    const double& fMean,
+    const bool population)
+{
+  double kurtosis = 0;
+  const double M4 = SumNthPowerDeviations(input, fMean, 4);
+  const double n = input.n_elem;
+  if (population)
+  {
+    // Calculate Population Excess Kurtosis
+    double M2 = SumNthPowerDeviations(input, fMean, 2);
```

Comment:
I think this is a good candidate of const too, as other variables in this function. 
Sorry of being picky on const correctness.

After this is fixed, I think the codes are ready to merge